lib/sisimai/bite/email/amazonses.rb

```diff
@@ -0,0 +1,247 @@
+module Sisimai::Bite::Email
+  module AmazonSES
+    # Sisimai::Bite::Email::AmazonSES parses a bounce email which created by
+    # Amazon Simple Email Service.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/AmazonSES.pm
+      require 'sisimai/bite/email'
+
+      # http://aws.amazon.com/ses/
+      ReE = { :'x-mailer' => %r/Amazon[ ]WorkMail/ }
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->